src/AppInstallerRepositoryCore/Microsoft/PreIndexedPackageSourceFactory.cpp

```diff
@@ -146,6 +147,8 @@ namespace AppInstaller::Repository::Microsoft
 
                 SQLiteIndex index = SQLiteIndex::Open(indexLocation.u8string(), SQLiteIndex::OpenDisposition::Immutable);
 
+                // We didn't use to store the source identifier, so we compute it here in case it's

```

Comment:
Strange comment with no code change.

Reply:
Since I added the identifier to the source details, it may look like it doesn't make sense to get the identifier here as `GetPackageFamilyNameFromDetails(details)` instead of as `details.Identifier`. But we need to do it because we didn't have the identifier in the sources file before, so it may be missing from `details`. I added the comment to clear it up for the future.